jlh/JonFirstApp/app/src/main/java/com/e_conomic/jonfirstapp/DisplayMessageFragment.java

```diff
@@ -1,17 +1,49 @@
 package com.e_conomic.jonfirstapp;
 
 import android.os.Bundle;
+import android.support.annotation.Nullable;
 import android.support.v4.app.Fragment;
 import android.view.LayoutInflater;
 import android.view.ViewGroup;
 import android.view.View;
+import android.widget.TextView;
+
 
 public class DisplayMessageFragment extends Fragment {
 
+    // The message to be displayed.
+    private String message = "";
+
+    // The TextView that displays the message.
+    private TextView display_message;
+
+    @Override
+    public void onCreate(@Nullable Bundle savedInstanceState) {
```

Comment:
I think so. It indicates that the parameter could be `null` so that you have to handle that case in the method body. Here, it's handled by the `super.onCreate(..)`. I guess it's like the optional parameters you have in Kotlin? :)